app/build.gradle

```diff
@@ -35,7 +35,7 @@ android {
         }
     }
     compileSdkVersion 27
-    buildToolsVersion "27.0.3"
+    buildToolsVersion '28.0.3'
```

Comment:
The travis-ci build fail is caused by build tool version
```
  - build-tools-27.0.3
  - android-27
```
you can also update the version to make a successful ci build